packages/adapter-postgres/package.json

```diff
@@ -25,8 +25,8 @@
   },
   "scripts": {
     "integration:local": "docker-compose up --abort-on-container-exit; docker-compose down",
-    "preintegration": "ts-node src/__integration__/setup.ts",
-    "integration": "jest --config=../../jest.config.integration.js"
+    "preintegration": "ts-node src/__integration__/setup.ts && ts-node src/__typo__/setup.ts",
+    "integration": "node --trace-warnings ../../node_modules/.bin/jest --runInBand --detectOpenHandles --config=../../jest.config.integration.js"
```

Comment:
Do you still need the extra flags here?

Reply:
Still need --runInBand for now, since the integration tests all share the same db.